src/libappimage/utils/ElfFile.cpp

```diff
@@ -0,0 +1,151 @@
+// system
+extern "C" {
+#include <fcntl.h>
+#include <unistd.h>
+#include <sys/mman.h>
+#include <sys/types.h>
+}
+
+// local
+#include "light_byteswap.h"
+#include "ElfFile.h"
+
+
+typedef Elf32_Nhdr Elf_Nhdr;
```

Comment:
Might want to like, "hide" `Elf_Nhdr`? Can it be "un-typedefed"?

Reply:
Done